catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPostIngestPlugin.java

```diff
@@ -181,7 +225,7 @@ private void process(ContentItem contentItem, List<ContentItem> contentItems) {
         return;
```

Comment:
Even if you decide to keep the instantiation of a  `ContentItem` for the nitf file, this debug message should probably be updated because the `contentItem.getFilename()` should probably be pulled from the `metacard `instead.